projects/canopy/src/lib/hero-img/hero-img-card-subtitle/hero-img-card-subtitle.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, HostBinding, ViewEncapsulation, Input } from '@angular/core';
+
+import type { HeadingLevel } from '../../heading';
+
+@Component({
+  selector: 'lg-hero-img-card-subtitle',
+  templateUrl: './hero-img-card-subtitle.component.html',
+  styleUrls: ['./hero-img-card-subtitle.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class LgHeroImgCardSubtitleComponent {
+  @HostBinding('class.lg-hero-img-card-subtitle') class = true;
```

Comment:
We usually have a unit test to make sure the CSS class was added correctly by the component, something like:

```
it('should have the default class', () => {
  expect(componentElement.getAttribute('class')).toContain('lg-hero-img-card-subtitle');
});
```

Are you okay to add that for this and other components too?